source/configuration/environments/infrastructure/nexus.rst

```diff
@@ -0,0 +1,21 @@
+=====
+Nexus
+=====
+
+Import custom certifications (optional)
+=======================================
+
+.. code-block:: yaml
+   :caption: environments/infrastructure/configuration.yml
+
+   nexus_api_url: "10.0.3.1:8190"
+   nexus_truststore_cert_urls:
+     - download.docker.com
+     - linux.mellanox.com
+
+Nexus rollout
+=============
+
+.. code-block:: console
+
+   osism-infrastructure nexus
```

Comment:
osism apply nexus?

Reply:
Let's still take the old commands. We need to change the whole documentation once to osism apply.